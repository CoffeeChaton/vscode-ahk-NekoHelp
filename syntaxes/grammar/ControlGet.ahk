ControlGet, OutputVar, List
ControlGet, OutputVar, Checked
ControlGet, OutputVar, Enabled
ControlGet, OutputVar, Visible
ControlGet, OutputVar, Tab
ControlGet, OutputVar, FindString
ControlGet, OutputVar, Choice
ControlGet, OutputVar, LineCount
ControlGet, OutputVar, CurrentLine
ControlGet, OutputVar, CurrentCol
ControlGet, OutputVar, Line
ControlGet, OutputVar, Selected
ControlGet, OutputVar, Style
ControlGet, OutputVar, ExStyle
ControlGet, OutputVar, Hwnd