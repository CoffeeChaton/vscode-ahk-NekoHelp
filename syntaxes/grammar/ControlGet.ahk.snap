>ControlGet, OutputVar, List
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Checked
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Enabled
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Visible
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Tab
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, FindString
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Choice
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, LineCount
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, CurrentLine
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, CurrentCol
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Line
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Selected
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Style
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, ExStyle
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk
>ControlGet, OutputVar, Hwnd
#^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk support.function.command.controlget.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.controlget.ahk
#                       ^^^^ source.ahk meta.command.line.controlget.ahk keyword.command.option.param.controlget.ahk